### PR TITLE
spaces

### PR DESCRIPTION
**Pull request by arghunter**

### CODE REVIEW

File: 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java [hunk 1]
```diff
@@ -139,12 +137,12 @@ public double getSpeed() {
 
     public double getAbsoluteEncoderRotations() {
         double pos = absEncoder.get();
-        while(pos > ABSOLUTE_ENCODER_POSITION_AT_FLIPOUT_MAX + 0.1){
-           pos -= 1;
+        while (pos > ABSOLUTE_ENCODER_POSITION_AT_FLIPOUT_MAX + 0.1) {
```
Comment (KatiaBravo) [inline 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java:140]: why is it inconsistent with the other while loop

File: 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java [hunk 2]
```diff
@@ -199,10 +198,9 @@ private static double convertFlipoutRotationsToDegrees(double rotations) {
     public void setSystemTargetAngleInDegrees(double angleDegrees) {
         angleDegrees = MathUtil.clamp(angleDegrees, 0, 90);
         double currentAngle = getCurrentAngleInDegrees();
-        if(angleDegrees > currentAngle){
+        if (angleDegrees > currentAngle) {
             rotator_controller.setSmartMotionMaxAccel(MAX_ACCEL_DOWNWARDS, this.SMARTMOTION_SLOT);
-        }
-        else{
+        } else {
```
Comment (KatiaBravo) [inline 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java:203]: yes.

File: 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java [hunk 3]
```diff
@@ -255,9 +253,9 @@ public void setRotatorMode(boolean coast) {
         }
     }
 
-    public boolean isDeployed(){
-        return (angle==90);
-    } 
+    public boolean isDeployed() {
```
Comment (KatiaBravo) [inline 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java:256]: debatable

**Review by KatiaBravo — CHANGES_REQUESTED** [review]
Needs a little help

File: 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java (hunk 1 above)
Comment (jhwang04) [inline 2022-Robot/src/main/java/frc/team670/robot/subsystems/Deployer.java:140]: again, why is this inconsistent with the other while loop?

File: 2022-Robot/src/main/java/frc/team670/robot/subsystems/ConveyorSystem.java [hunk 4]
```diff
@@ -28,7 +28,6 @@ public enum Status {
 	private Status status = Status.OFF;
 	private Timer timer = new Timer();
 	private final int CONVEYOR_IDLE_CHECK_PERIOD = 2;
-
```
Comment (KatiaBravo) [inline 2022-Robot/src/main/java/frc/team670/robot/subsystems/ConveyorSystem.java:31]: slay

**Review by KatiaBravo — APPROVED** [review]
good to go

**Pull request merged**